src/Servant/Server/Internal.hs

```diff
@@ -160,7 +163,7 @@ instance (FromHttpApiData a, HasServer sublayout context m)
 
   route Proxy ctx d =
     CaptureAllRouter $
-        route (Proxy :: Proxy sublayout) ctx
+        route (Proxy @sublayout) ctx
```

Comment:
Don't. This is inconsistent with all other use in `servant` codebase.

Also In this case these kind of changes increase overall diff size. The bigger the diff, the longer it will take to review.

Reply:
Thanks, will revert!